target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -80,6 +80,9 @@ comfast,cf-e560ac)
 	ucidef_set_led_switch "lan3" "LAN3" "$boardname:blue:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "$boardname:blue:lan4" "switch0" "0x10"
 	;;
+comfast,cf-wr752ac-v1)
+  ucidef_set_led_netdev "lan" "LAN" "$boardname:blue:lan" "eth0"
+  ;;
```

Comment:
Wrong indent, despite this can be merged with ecb1750 ...

Reply:
merged with ecb1750